scripts/backup

```diff
@@ -28,7 +28,14 @@ ynh_backup --src_path="$install_dir"
 ynh_backup --src_path="$data_dir" --is_big
 
 #=================================================
-# SYSTEM CONFIGURATION
+# BACKUP THE NGINX CONFIGURATION
+#=================================================
+
+ynh_backup --src_path="/etc/nginx/conf.d/$domain.d/$app.conf"
+ynh_backup --src_path="/etc/nginx/conf.d/$domain.d/$app.d"
+
+#=================================================
+# BACKUP THE PHP-FPM CONFIGURATION
```

Comment:
Why this section ? Just put everything in SYSTEM CONFIGURATION just after line 45 :)